### DISCUSSION TITLE
i try to login and i get this error: An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by iamyahia** [vercel/community#3315] [Help] [answered]
### Summary

Hello everyone. When I try to log into Vercel using github, I receive the following error: After entering my phone number, the message "An unexpected internal problem occurred" appeared, as shown below.
![loginprr](https://github.com/vercel/community/assets/90482598/29cef36d-0e1f-432a-aa7e-c21066fa473e)
 and before in the past i have a hosted  

### Steps to Reproduce

just try to login to vercel in [login page](https://vercel.com/signup)

**Comment by amyegan** [comment, marked as answer]
Hey @iamyahia. I think your login problem is related to this issue: https://www.vercel-status.com/incidents/tp22kxv7rfcm

**Reply by iamyahia** [reply]
ouu you right @amyegan  I'll subscribe the issue.
thank you so much for your care.

**Reply by amyegan** [reply]
Happy to help! 😄 

If you continue to have login problems after the related issue is resolved, please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by iamyahia** [comment]
after vercel announce that the system is normal,
i try to signup/signin with my github account but it gives me the same error !!!

please what happen ?? how can i signin/signup to vercel ??

@amyegan can you help please??
